### PR TITLE
lzma-native package updated to latest

### PR DESCRIPTION
**Pull request by rahul606**

### CODE REVIEW

**Comment by addaleax** [comment]
Just fyi, there’s a typo in the commit message (`lama`) :smile:

**Comment by jviotti** [comment]
Very nice @rahul606 , thanks a lot! I'll merge as soon as you fix the commit title!

BTW, is this the reason behind https://github.com/resin-io/etcher/issues/981, or just a miscellaneous improvement you saw after digging into the module?

**Comment by rahul606** [comment]
@jviotti I did the change for comment, but I am not sure if everything went fine or not. Editing the comment on github is new to me as I usually do this via IDE.

This problem did not solved resin-io/etcher#981 problem but it did solve the problem of running CLI correctly on mac.

**Comment by jviotti** [comment]
The problem of the CLI not running on your Mac is that you previously compiled the native dependencies against the version of Electron we build Etcher for, and not against your locally installed version of NodeJS.

When you did `npm install lzma-native@latest` you forced npm to re-install that specific native package against your NodeJS installation, which made it all work, but there is no problem at all there. If you run `npm install --build-from-source` in the Etcher repository, it will fetch the current version of `lzma-native`, but will build it (or download a pre-built binary) agains your local NodeJS version.

So in summary, the version doesn't make any difference (its the way you install the dependencies), but I'll take this PR anyway!
